### ISSUE TITLE
Steammoded 1.0 compatible?

### ISSUE THREAD

**Issue by OmarHonigman** [PGgamer2/balatro-mobile-builder#6]
"Is it possible to make it compatible with Steammoded 1.0? Using Balatro Mobile Maker, I managed to run mods that use Steammoded 0.9.8, but those using newer versions crash. So, is it possible to use version 1.0?"

**Comment by PGgamer2** [comment]
Steamodded first needs for Lovely to add Android and iOS support. There's currently an issue on their repo https://github.com/ethangreen-dev/lovely-injector/issues/32

**Comment by DracyanZeutron** [comment]
yes it is possible to make it compatible but you need to twinker a little i might help you if you like it @OmarHonigman

**Comment by Judipie** [comment]
@DracyanZeutron i could really use some help with this, if you can that would be awesome!
